src/App.js

```diff
@@ -4,9 +4,25 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  handlerChange = (event) => {
```

Comment:
```suggestion
  handleChange = (event) => {
```
Pay attention to naming - `handleChange` (what to do?) or `changeHandler` (what is it?)